internal/pkg/task/env_runner_test.go

```diff
@@ -30,6 +30,10 @@ func TestEnvRunner_Run(t *testing.T) {
 			Values: []string{inApp},
 		},
 	}
+	filtersForSecurityGroup := append(filtersForSubnetID, ec2.Filter{
+		Name:   fmt.Sprintf(ec2.TagFilterName, envSecurityGroupCFNLogicalIDTagKey),
```

Comment:
nit: Can we use the strings directly? like "tag:aws:cloudformation:logical-id" to make sure that the function under test does the translation properly